challenge_1/javascript/Zirhc86/src/stringReverse.js

```diff
@@ -0,0 +1,25 @@
+
+console.log('Running test() with no argument will log \'Example string!\' reversed.');
+
+/*
+ ********************************************************************
+ ************************CHALLENGE ANSWER ***************************
```

Comment:
This isn't necessary. Let your code speak as the solution, rather you speaking for us.